.circleci/config.yml

```diff
@@ -430,6 +431,60 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/test-reports/report.htm
 
+  integration_test_ios_fennec:
+    macos:
+      xcode: 15.3.0
+    steps:
+      - checkout
```

Comment:
Add the path checks

      - check_file_paths:
          paths: "experimenter/"